uplink/src/lib.rs

```diff
@@ -195,9 +191,8 @@ pub mod config {
                     topic: format!(
                         "/tenants/{tenant_id}/devices/{device_id}/events/{stream_name}/jsonarray"
                     ),
-                    buf_size: config.system_stats.stream_size.unwrap_or(100),
```

Comment:
Are these changes necessary in this PR?

Reply:
`unwrap_or` is changed to `MAX_BUFFER_SIZE`. Looks unrelated and no idea what that value is